apps/ledger-live-desktop/src/renderer/analytics/segment.ts

```diff
@@ -145,6 +152,7 @@ const extraProperties = (store: ReduxStore) => {
     hasGenesisPass,
     hasInfinityPass,
     modelIdList: devices,
+    stakingProvidersEnabled: stakingProviders?.length || "flag not loaded",
```

Comment:
Are we sure we want to assign a number or a string to the same variable ?

Reply:
you are correct, I am going to merge this just because it needs to be available for the release but I will submit a refactor PR shortly